### ISSUE TITLE
Found a way to bypass IP ban!

### ISSUE THREAD

**Issue by DjayColes** [finndev/PokeBuddy#936]
So last night i kept getting IP banned using Pokebuddy and noticed it was my wifi, so i turned wifi off and all the pokestops reappeared etc. On my iphone i downloaded an app call Cyberghost, a vpn app to hide what you're doing, I then connected my laptop to my phones 4G via hotspot and the botted overnight with the laptop still botting when i woke up. I have an unlimited data plan so works out sweet for me!

**Comment by JewBagel** [comment]
cyberghost isnt free and you switching to 4g switched your ip regardless so using a vpn was pointless in your case

**Comment by DjayColes** [comment]
well it may work without Cyberghost but I'm yet to try it, just saying what I done that's all...

**Comment by JewBagel** [comment]
yeah i get it but its common sense to anyone with simple computer knowledge

**Comment by DjayColes** [comment]
well okay lets just forget i used the cyberghost then, im not familiar with VPN's and how they work, sorry for trying to help others jeeez

**Comment by JewBagel** [comment]
a vpn makes your computer appear like its in a different place changing your ip address which niantic was softbanning ips yesterday now there getting even deeper into banning more seriously it seems

**Comment by DjayColes** [comment]
My PokemonGo app is at my actual location so I hotspotted to my 4G and botted at my real location also, seems to have worked fine for now, I'l bot a bit more and teleport to see if it still works, i very much doubt it will work though

**Comment by JewBagel** [comment]
my bots not working at all right now id be surprised if it works for you let me know

**Comment by etowery22** [comment]
Be very careful with 3rd party VPN's. You are actually authenticating to their servers and sending all of your data through their access points. In other words, they get to see and capture all of your data that you are transmitting.
